src/fidesops/core/config.py

```diff
@@ -41,6 +41,7 @@ class ExecutionSettings(FidesSettings):
     MASKING_STRICT: bool = True
     CELERY_BROKER_URL: str = "redis://:testpassword@redis:6379/1"
     CELERY_RESULT_BACKEND: str = "redis://:testpassword@redis:6379/1"
+    USE_DEDICATED_WORKER: bool = True
```

Comment:
For clarity, I'd recommend this being `WORKER_ENABLED` and it defaults to true, which means by _default_ we have that worker running. Then the advanced mode is you set `WORKER_ENABLED` to false, and you need to run a second node?

Reply:
Thanks! Good suggestion